### ISSUE TITLE
FR: minify statically cached files

### ISSUE THREAD

**Issue by lukeagsmith** [octoper/statamic-html-minify#8]
The add-on is great, but when I turned static caching the html created was no-longer minified.

Is is possible to make the add-on work with static caching?

**Comment by octoper** [comment]
I tried to reproduce the issue but I couldn't it's working fine for me.

**Comment by lukeagsmith** [comment]
Thanks for coming back to me Vaggelis.

So for you, when you directly open the files that get written to the `static` folder, they are minified html?

**Comment by octoper** [comment]
You are right they are not minified but when I'm browsing the page it is working just fine.
So I don't think thats a problem with the plugin but with Statamic I think it's applying the middleware after it's generating the static page.

**Comment by hlorofos** [comment]
It doesn't work for me either. Saved static files aren't minified
